packages/datagateway-search/src/search/searchButton.component.test.tsx

```diff
@@ -52,7 +76,31 @@ describe('Search Button component tests', () => {
     expect(wrapper).toMatchSnapshot();
   });
 
-  it('builds correct parameters for request if date and search text properties are in use', () => {
+  it('builds correct parameters for datafile request if date and search text properties are in use', () => {
+    state.dgsearch = {
```

Comment:
These tests can be simplified by using object spread and only specifying the state changes needed for the specific tests, e.g.

```
state.dgsearch = {
    ...state.dg.search,
    searchText: "hello",
    selectDate: {
        startDate: new Date('2013-11-11'),
        endDate: new Date('2016-11-11'),
      },
    checkBox: {
         ...state.dg.search.checkbox,
         datafile: true 
    },
}
```